Doc/library/asyncio-stream.rst

```diff
@@ -201,6 +201,16 @@ StreamWriter
 
       Close the transport: see :meth:`BaseTransport.close`.
 
+   .. coroutinemethod:: wait_closed()
+
+      Wait for writer closing.
+
+      Should be called after :meth:`close` to waiting for finishing
+      all writer activities (:meth:`BaseProtocol.connection_lost`
+      callback call actually).
```

Comment:
-> "Should be called after :meth:`close` to wait until the underlying
connection (and the associated transport/protocol pair) is closed."